ghcup-0.0.7.yaml

```diff
@@ -3610,6 +3609,101 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~ghc/9.4.7/ghc-9.4.7-aarch64-apple-darwin.tar.xz
               dlSubdir: ghc-9.4.7-aarch64-apple-darwin
               dlHash: 5d85f9836d72d45634039218ed52e9faa0ed00c0db056f3d1162b4c2b3838e38
+    9.4.8:
+      viTags:
+      - base-4.17.2.1
+      - Recommended
+      viChangeLog: https://downloads.haskell.org/~ghc/9.4.8/docs/users_guide/9.4.8-notes.html
+      viSourceDL:
+        dlUri: https://downloads.haskell.org/~ghc/9.4.8/ghc-9.4.8-src.tar.xz
+        dlSubdir: ghc-9.4.8
+        dlHash: 06775a52b4d13ac09edc6dabc299fd11e59d8886bbcae450af367baee2684c8f
+      viTestDL:
+        dlUri: https://downloads.haskell.org/ghc/9.4.8/ghc-9.4.8-testsuite.tar.xz
+        dlSubdir: ghc-9.4.8/testsuite
+        dlHash: a7ce907f7e3d38bee33af5cc23c74b53fe30d600fe5fc63b409c4c5675d2a69d
+      viArch:
+        A_64:
+          Linux_Debian:
+            '< 10': &ghc-948-64-deb9
+              dlUri: https://downloads.haskell.org/~ghc/9.4.8/ghc-9.4.8-x86_64-deb9-linux.tar.xz
+              dlSubdir: ghc-9.4.8-x86_64-unknown-linux
+              dlHash: b9d2b4326c907c5ce5fe0502ea089583c1af4529a17f2e18d103f10f430bb036
+            '(>= 10 && < 11)': &ghc-948-64-deb10
+              dlUri: https://downloads.haskell.org/~ghc/9.4.8/ghc-9.4.8-x86_64-deb10-linux.tar.xz
+              dlSubdir: ghc-9.4.8-x86_64-unknown-linux
+              dlHash: fc77eaae5b89f29177bf159fd95ce438066ec64a46bf69df61b267102afdb10e
+            '>= 11': &ghc-948-64-deb11
+              dlUri: https://downloads.haskell.org/~ghc/9.4.8/ghc-9.4.8-x86_64-deb11-linux.tar.xz
+              dlSubdir: ghc-9.4.8-x86_64-unknown-linux
+              dlHash: 2743629d040f3213499146cb5154621d6f25e85271019afc9b9009e04d66bf6c
+            unknown_versioning: *ghc-948-64-deb11
```

Comment:
In the recent cabal PR discussion, we decided it's probably best to fallback to the oldest bindist in case we can't figure out a version (in this case `ghc-948-64-deb9`).